### PR TITLE
IP Messenger v5.1.0

### PR DESCRIPTION
**Pull request by shirouzu**
- [✓] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [✓] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [✓] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [✓] Have you tested your manifest locally with `winget install --manifest <path>`?
- [✓] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?
-----
IP Messenger v5.1.0 released.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/29487)

### CODE REVIEW

File: manifests/f/FastCopy/IPMsg/5.1.0/FastCopy.IPMsg.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+PackageIdentifier: FastCopy.IPMsg
+PackageVersion: 5.1.0
+PackageName: IP Messenger
```
Comment (ItzLevvie) [inline manifests/f/FastCopy/IPMsg/5.1.0/FastCopy.IPMsg.yaml:3]:
```suggestion
PackageName: IP Messenger for Win
```

File: manifests/f/FastCopy/IPMsg/5.1.0/FastCopy.IPMsg.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+Publisher: FastCopy Lab, LLC.
```
Comment (ItzLevvie) [inline manifests/f/FastCopy/IPMsg/5.1.0/FastCopy.IPMsg.yaml:4]:
```suggestion
Publisher: H.Shirouzu & FastCopy Lab, LLC.
```

Comment (ItzLevvie) [reply manifests/f/FastCopy/IPMsg/5.1.0/FastCopy.IPMsg.yaml:4]: Windows Package Manager requires `Publisher` to match Control Panel.

File: manifests/f/FastCopy/IPMsg/5.1.0/FastCopy.IPMsg.yaml (hunk 1 above)
Comment (ItzLevvie) [reply manifests/f/FastCopy/IPMsg/5.1.0/FastCopy.IPMsg.yaml:3]: Windows Package Manager requires `PackageName` to match Control Panel.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by shirouzu — COMMENTED** [review]
reflect suggestions.

**Review by ItzLevvie — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**